<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNCfileDlg</class>
 <widget class="QDialog" name="CreateNCfileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成nc</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listPaths"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addPath">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset resource="Resources/Custom.qrc">
             <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePath">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
           <property name="icon">
            <iconset resource="Resources/Custom.qrc">
             <normaloff>:/icons/multiply.png</normaloff>:/icons/multiply.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upPath">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>上移</string>
           </property>
           <property name="icon">
            <iconset resource="Resources/Custom.qrc">
             <normaloff>:/icons/Arrow_up.png</normaloff>:/icons/Arrow_up.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downPath">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>下移</string>
           </property>
           <property name="icon">
            <iconset resource="Resources/Custom.qrc">
             <normaloff>:/icons/Arrow_down.png</normaloff>:/icons/Arrow_down.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>机床坐标系</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="editSafeHeight"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="editWorkPos">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>位置</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>旋转</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editMachineRotate">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="editWorkRotate">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>工件坐标系</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>旋转</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editMachinePos">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>安全高度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>位置</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>NC 路径执行前
的附加机代码</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPlainTextEdit" name="editBeforeGCode"/>
       </item>
       <item row="5" column="3">
        <widget class="QPlainTextEdit" name="editAfterGCode"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>输出位置</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>NC 路径执行后
的附加机代码</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>转速</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLineEdit" name="editSpeed"/>
       </item>
       <item row="7" column="1" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="outputFilePath"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnOpenFileDialg">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources/Custom.qrc"/>
 </resources>
 <connections/>
</ui>
